build.sbt

```diff
@@ -17,6 +17,8 @@ libraryDependencies ++= Seq(
   "io.gatling" % "gatling-test-framework" % "2.2.5" % Provided
 )
 
+parallelExecution in Test := false
```

Comment:
Is this still a problem?

Reply:
Had to modify docker-compose file to use map random ports and change some test validations.
But now tests can be run in parallel.